README.md

```diff
@@ -1 +1,32 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+This App is a Game Leaderboard app Which is created by JavaScript and the big picture of this application is using API.
+
```

Comment:
- [ ] You have a good README file but I think it would be better to update your README file by adding how to get and set your project locally, kindly check this Microverse README [template](https://github.com/microverseinc/readme-template) to update your with all the missing steps of your README file